app/models/user.rb

```diff
@@ -26,7 +26,7 @@ def new_token
 
   def remember
     self.remember_token = User.new_token
-    update_attributes :remember_digest, User.digest(remember_token)
+    update_attribute :remember_digest, User.digest(remember_token)
```

Comment:
nếu để "update_attributes :remember_digest, User.digest(remember_token)"
 nó báo lỗi "wrong number of arguments (given 2, expected 1)"